<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="qawin7qa01" name="com.cat.AutomationPage.TextAndGraphicTemplateScript" tests="8" failures="0" timestamp="16 Feb 2018 13:42:23 GMT" time="175.189" errors="2">
  <testcase name="CatEditTextTemplate" time="90.880" classname="com.cat.AutomationPage.TextAndGraphicTemplateScript">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.cat.utility.WebAppCommon.waitForElementPresentByID(WebAppCommon.java:1359)
	at com.cat.utility.WebAppCommon.typeTextById(WebAppCommon.java:1551)
	at com.cat.AutomationScript.TextAndGraphicScriptUtili.CSSAdd(TextAndGraphicScriptUtili.java:715)
	at com.cat.AutomationScript.TextAndGraphicScriptUtili.editTextTemplate(TextAndGraphicScriptUtili.java:662)
	at com.cat.AutomationPage.TextAndGraphicTemplateScript.CatEditTextTemplate(TextAndGraphicTemplateScript.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- CatEditTextTemplate -->
  <testcase name="CatLogicTC" time="20.695" classname="com.cat.AutomationPage.TextAndGraphicTemplateScript"/>
  <testcase name="CatAddTopic" time="4.698" classname="com.cat.AutomationPage.TextAndGraphicTemplateScript"/>
  <testcase name="CatCreateCourseTC" time="33.781" classname="com.cat.AutomationPage.TextAndGraphicTemplateScript"/>
  <testcase name="CatAddLesson" time="7.043" classname="com.cat.AutomationPage.TextAndGraphicTemplateScript"/>
  <testcase name="CatAddPage" time="4.683" classname="com.cat.AutomationPage.TextAndGraphicTemplateScript"/>
  <testcase name="CatResetTest" time="13.409" classname="com.cat.AutomationPage.TextAndGraphicTemplateScript"/>
  <testcase name="CatLogOutTC" time="0.000" classname="com.cat.AutomationPage.TextAndGraphicTemplateScript">
    <error type="java.lang.Throwable" message="Method TextAndGraphicTemplateScript.CatLogOutTC()[pri:0, instance:com.cat.AutomationPage.TextAndGraphicTemplateScript@43738a82] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method TextAndGraphicTemplateScript.CatLogOutTC()[pri:0, instance:com.cat.AutomationPage.TextAndGraphicTemplateScript@43738a82] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- CatLogOutTC -->
</testsuite> <!-- com.cat.AutomationPage.TextAndGraphicTemplateScript -->
